rust/tool/shared/Cargo.toml

```diff
@@ -18,3 +18,14 @@ sha2 = "0.10"
 # different versions.
 fastrand = "2.0.1"
 log = { version = "0.4", features = ["std"] }
+serde = { version = "1.0.194", features = ["derive"] }
+stderrlog = "0.5"
+indoc = "2"
+ureq = { version = "2.7.1", features = [ "json" ] }
```

Comment:
This should be an optional depedency guarded by a compile time flag.

Reply:
Right.

Reply:
I did something, let me know how you like it.